src/main/java/jenkinsci/plugins/influxdb/generators/SonarQubePointGenerator.java

```diff
@@ -80,21 +84,30 @@ public SonarQubePointGenerator(Run<?, ?> build, TaskListener listener,
                                    ProjectNameRenderer projectNameRenderer,
                                    long timestamp,
                                    String jenkinsEnvParameterTag,
-                                   String customPrefix) {
+                                   String customPrefix,
+                                   EnvVars env) {
         super(build, listener, projectNameRenderer, timestamp, jenkinsEnvParameterTag);
         this.customPrefix = customPrefix;
         this.listener = listener;
+        this.env = env;
     }
 
+    /**
+     * @return true, if environment variable LOG_SONAR_QUBE_RESULTS is set to true and SQ Reports exist
+     */
     public boolean hasReport() {
-        try {
-
-            String[] result = getSonarProjectURLFromBuildLogs(build);
+        String logMessage = "[InfluxDB Plugin] SonarQube hasReport";
+        listener.getLogger().println(logMessage);
```

Comment:
This seems like an irrelevant log message outside development debugging. If SonarQube report is found it will log `SonarQube data found. Writing to InfluxDB...` anyway in `InfluxDbPublicationService.java`.